CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
```

Comment:
Since it is just a dev change which does not effect the final user, we don't need a changelog entry.

Here you can find when a new entry is needed or not: https://github.com/scanapi/scanapi/wiki/Changelog#what-warrants-a-changelog-entry

Reply:
Fixed in 11c2650